playwright/click-button.spec.ts

```diff
@@ -1,14 +1,20 @@
 import { test } from '@playwright/test';
 
 test.describe('Buttons navigation tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+  });
+
   async function clickAndVerifySection(page, elementLocator): Promise<void> {
     await page.click(elementLocator);
     await page.locator('[data-testid="auth-section"]').boundingBox();
```

Comment:
i dont know what is this string in locator, need to move it to const and add correct name which will explain what is this